firmware/common/ais_packet.hpp

```diff
@@ -42,6 +42,7 @@ struct DateTime {
 
 template<size_t FieldSize, int32_t DegMax, uint32_t NAValue>
 struct LatLonBase {
+	
```

Comment:
When you do a diff (in Github desktop for example) you can remove this small changes in style. It is fine, but just keep it in mind (i.e. in this case 1 line would be cleaner for the commit)